docs/spec/spec.md

```diff
@@ -121,6 +122,31 @@ Subscription's call parameter._
 
 ---
 
+## kind: ChannelSubscriptionSet
+
+### group: eventing.internal.knative.dev/v1alpha1
+
+_A ChannelSubscriptionSet holds an aggregation of resolved subscriptions for a
+Channel._
+
+### Object Schema
+
+#### Spec
+
+| Field         | Type                    | Description                                                           | Limitations                            |
+| ------------- | ----------------------- | --------------------------------------------------------------------- | -------------------------------------- |
+| subscribers\* | ChannelSubscriberSpec[] | Information about subscriptions used to implement message forwarding. | Filled out by Subscription Controller. |
```

Comment:
To clarify what Subscription controller actually does:
1. Subscription controller reconciles a Subscription.
1. If the Subscription names a Channel in its `from`, Subscription controller gets that Channel then gets the ChannelSubscriptionSet in its Status.
1. Subscription controller looks for a subscriber in ChannelSubscriptionSet matching the Subscription being reconciled, and adds one if none is found. (There's an issue here around losing a reference to the original subscription, so Subscription updates don't work and duplicate subscriptions aren't possible, but that's an issue with the current spec also.)

Is this correct?

Reply:
I lifted this line from the current [Channelable definition](https://github.com/n3wscott/eventing/pull/5/files/19ebafb9283c2122724bbe441fbd1d0e32decfeb#diff-bd2ff29d22d19f9b6c62779fda3647d8L309), the only change was to make the field required.

What you listed is the intended behavior. I thought I had also stated this explicitly. I'll add a clarification.